### ISSUE TITLE
flyctl platform status --json opens a web browser.

### ISSUE THREAD

**Issue by gedw99** [superfly/flyctl#1971]
```
fly version
fly v0.0.500 darwin/amd64 Commit: f71fe0ef BuildDate: 2023-03-29T16:56:42Z
flyctl platform status --json 
```

So that we can monitor using tooling please. Opening a web page is not helpful.

**Comment by alichay** [comment]
Fixed in version `0.0.505`, thanks for reporting!

**Comment by gedw99** [comment]
thanks you @alichay 

works a treat :)
